test/suites/backup.sh

```diff
@@ -507,15 +507,21 @@ test_backup_rename() {
 }
 
 test_backup_volume_export() {
-  test_backup_volume_export_with_project
-  test_backup_volume_export_with_project fooproject
+  test_backup_volume_export_with_project default false
+  test_backup_volume_export_with_project fooproject false
+
+  if [ "$lxd_backend" = "ceph" ] && [ -n "${LXD_CEPH_CEPHFS:-}" ]; then
+    test_backup_volume_export_with_project default true
+    test_backup_volume_export_with_project fooproject true
+  fi
 }
 
 test_backup_volume_export_with_project() {
-  project="default"
   pool="lxdtest-$(basename "${LXD_DIR}")"
+  project="$1"
+  use_cephfs="$2"
```

Comment:
I would suggest calling this argument's var `custom_vol_pool`, and then always pass in the pool name to it.

Then in  `test_backup_volume_export()` you would pass the 2nd argument as `"lxdtest-$(basename "${LXD_DIR}")"` and then in the cephfs section you would create the cephfs storage pool and then pass the 2nd argument as the name of that pool.

That way it keeps the cephfs specific logic in one place.

Reply:
It's fixed now. I stuck with the name `pool` though as I don't see the necessity of renaming it.